CHANGELOG.md

```diff
@@ -22,9 +22,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixes
 - Fix wrong typings generated for dist @kuzhelov ([#99](https://github.com/stardust-ui/react/pull/99))
+- Fix `icon` shorthand property for Button @kuzhelov ([#112](https://github.com/stardust-ui/react/pull/112))
 
 ### Features
 - Add Menu `iconOnly`, MenuItem `iconOnly` and `icon` props @miroslavstastny ([#73](https://github.com/stardust-ui/react/pull/73))
+- Add conformance tests for component shorthand properties @kuzhelov ([#112](https://github.com/stardust-ui/react/pull/112))
```

Comment:
Note, we only add entries for public facing changes.  Chores, like tests, build, and CI changes do not get changelog entries as they do not affect end users.